common/lib/xmodule/xmodule/xml_module.py

```diff
@@ -428,6 +429,13 @@ def add_xml_to_node(self, node):
         """
         # Get the definition
         xml_object = self.definition_to_xml(self.runtime.export_fs)
+        for aside in self.runtime.get_asides(self):
+            if aside.needs_serialization():
+                aside_node = etree.Element("unknown_root", nsmap=XML_NAMESPACES)
+                aside.has_children = False
```

Comment:
This seems odd. `has_children` has typically been a class attribute on XBlocks. Perhaps we need it set at the top level of the XBlockAside base-class? Or perhaps the code that failed when you left this out should be using a `getattr` with a default?